tests/integration/targets/aci_l3out_static_routes_nexthop/tasks/main.yml

```
@@ -109,6 +111,18 @@
       <<: *nh_present
       nexthop: 1.1.1.2
 
+  - name: Add another nexthop without preference
+    cisco.aci.aci_l3out_static_routes_nexthop: 
+      <<: *prefix_present
+      nexthop: 2.2.2.2
+    register: add_nh_no_pref
+
+  - name: Verify default preference
+    ansible.builtin.assert:
+      that:
+      - add_nh_no_pref.is_changed
```

is there a reason to use is_changed? could you keep consistent with the rest?

```suggestion
      - add_nh_no_pref is changed
```

---

Thank you for your confirmation. It's my mistake. I will correct it as you pointed out. excuse me.